module/VuFind/src/VuFind/View/Helper/Root/HelpText.php

```diff
@@ -38,26 +38,12 @@
  */
 class HelpText extends \Laminas\View\Helper\AbstractHelper
 {
-    /**
-     * The current language
-     *
-     * @var string
-     */
-    protected $language;
-
-    /**
-     * The default fallback language
-     *
-     * @var string
-     */
-    protected $defaultLanguage;
-
     /**
      * The context view helper
```

Comment:
This should be changed to `Content` as well.

Reply:
done!